cdap-ui/app/cdap/components/Home/index.js

```diff
@@ -108,8 +113,9 @@ export default class Home extends Component {
           <Route path="/ns/:namespace/experiments" component={Experiments} />
           <Route exact path="/ns/:namespace/operations" component={OpsDashboard} />
           <Route exact path="/ns/:namespace/details" component={NamespaceDetails} />
-          <Route exact path="/ns/:namespace/create-profile" component={ProfileCreateView} />
           <Route path="/ns/:namespace/reports" component={Reports} />
+          <Route exact path="/ns/:namespace/profiles/create" component={ProfileCreateView} />
+          <Route exact path="/ns/:namespace/profiles/details/:profileId" component={ProfileDetailView} />
```

Comment:
Shouldn't this be `/ns/:namespace/profiles/:profileId/details`? just like the namespace details url

Reply:
Talked to @tonybach we are going with `/profiles/details/:profileid`. if it is the other way we can't have `/profiles/create` which is for creating a profile in UI.